Sources/SourceKitLSP/CapabilityRegistry.swift

```diff
@@ -0,0 +1,103 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2021 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import LanguageServerProtocol
+
+/// Handler responsible for registering a capability with the client.
+public typealias RegistrationHandler = (_ registration: CapabilityRegistration) -> Void
```

Comment:
Maybe `ClientRegistrationHandler`?

Nitpick: I don't think the "_ registration:" adds anything to this typealias.

Reply:
Done